OpenKh.Tools.ModsManager/Services/ConfigurationService.cs

```diff
@@ -51,8 +51,9 @@ public static Config Open(string fileName)
             }
         }
 
-        private const string ConfigPath = "mods-manager.yml";
-        private const string EnabledModsPath = "mods.txt";
+        private static string _assString = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+        private static string ConfigPath = _assString + "\\mods-manager.yml";
```

Comment:
I highly suggest to use `Path.Combine` rather than `+ "\\"` as it internally covers weird scenario that might occur. Same for the lines below.

Reply:
Can do.